File: sdkwork-backend-sdk-go/types/group_info.go
package types


type GroupInfo struct {
	Id string `json:"id"`
	Object string `json:"object"`
	Created int `json:"created"`
	Model string `json:"model"`
	SystemFingerprint string `json:"systemFingerprint"`
	Metadata ChatCompletionMetadata `json:"metadata"`
	Agent PlusAgent `json:"agent"`
}

File: sdkwork-backend-sdk-csharp/Models/GroupInfo.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class GroupInfo
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public int? Created { get; set; }
        public string? Model { get; set; }
        public string? SystemFingerprint { get; set; }
        public ChatCompletionMetadata? Metadata { get; set; }
        public PlusAgent? Agent { get; set; }
    }
}
